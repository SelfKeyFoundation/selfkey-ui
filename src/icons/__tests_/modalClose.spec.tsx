import * as React from 'react';
import { create } from 'react-test-renderer';
import ModalCloseIcon from '../modal-close';

describe('Modal Close Icon', () => {
	it('should show Modal Close Icon', () => {
		const component = create(<ModalCloseIcon />);
		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
