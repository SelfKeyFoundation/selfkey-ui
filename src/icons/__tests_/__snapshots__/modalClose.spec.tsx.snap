// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal Close Icon should show Modal Close Icon 1`] = `
<svg
  aria-hidden="true"
  className="MuiSvgIcon-root Component-root-0-2-1 MuiSvgIcon-fontSizeLarge"
  focusable="false"
  height="38px"
  viewBox="0 0 38 38"
  width="38px"
>
  <g
    fill="none"
    fillRule="evenodd"
    id="Symbols"
    stroke="none"
    strokeWidth="1"
  >
    <g
      id="Close-button"
      transform="translate(1.000000, 1.000000)"
    >
      <g
        id="icon-close-modal"
      >
        <circle
          cx="18"
          cy="18"
          fill="#262F39"
          id="Oval-2"
          r="18"
        />
        <g
          fill="#23E6FE"
          fillRule="nonzero"
          id="Shape"
          transform="translate(9.000000, 9.000000)"
        >
          <polygon
            points="1.81534091 0.639204545 0.639204545 1.81534091 7.82386364 9 0.639204545 16.1846591 1.81534091 17.3607955 9 10.1761364 16.1846591 17.3607955 17.3607955 16.1846591 10.1761364 9 17.3607955 1.81534091 16.1846591 0.639204545 9 7.82386364"
          />
        </g>
      </g>
    </g>
  </g>
</svg>
`;
